src/components/Homepage/ThemeSection.tsx

```diff
@@ -177,7 +177,7 @@ export const ThemeSection = ({
                   Discovery
                 </Heading>
                 <m.div
-                  className="su-w-fit su-border-t-[2.8rem] su-border-bottom-none su-overflow-hidden"
+                  className="su-w-fit su-border-t-[2.6rem] su-border-bottom-none su-overflow-hidden"
```

Comment:
Normalizing the thickness of the color bars. These are the red triangles that morph into bars in the Homepage theme section. Just changing up the heights here so when it turns into a bar, it uses one of the set bar thickness (26px)